### PR TITLE
Add command to update installed browsers

### LINKED ISSUE #16: Add option to update browsers
We don't really have a good centralized place to manage the installed browsers used to create programmatic recordings --- these are used by our playwright, puppeteer, and cypress forks.  While this package has some functionality exposed to other node packages for managing the installed browsers, I think we should expose this in the CLI as well so that users can run `replay-recordings --update-browsers` to make sure that the browsers they are using are all installed and updated to the latest version.

### PR DESCRIPTION
**Pull request by bhackett1024**
Fixes https://github.com/RecordReplay/recordings-cli/issues/16

### CODE REVIEW

File: src/install.js [hunk 1]
```diff
@@ -38,27 +51,53 @@ function getPlaywrightBrowserPath(kind) {
...
+  const replayDir = getDirectory(opts);
+  const browserDir = path.join(replayDir, subdir);
+  const dstDir = path.join(browserDir, dstName);
+
+  if (fs.existsSync(dstDir)) {
+    // Remove the browser so installReplayBrowser will reinstall it. We don't have a way
+    // to see that the current browser is up to date.
+    fs.rmSync(dstDir, { force: true, recursive: true });
+  } else {
+    return;
```
Comment (ryanjduffy) [inline src/install.js:90]: Let's report an error here rather than stopping silently.

File: src/main.js [hunk 2]
```diff
@@ -404,9 +405,11 @@ module.exports = {
   viewLatestRecording,
   removeRecording,
   removeAllRecordings,
+  updateBrowsers,
 
   // These methods aren't documented or available via the CLI, and are used by other
   // @recordreplay NPM packages.
   ensurePlaywrightBrowsersInstalled,
   getPlaywrightBrowserPath,
+  updateBrowsers,
```
Comment (ryanjduffy) [inline src/main.js]: This line isn't needed

**Review by ryanjduffy — APPROVED** [review]
Couple minor notes but otherwise :+1:

File: src/install.js (hunk 1 above)
Comment (bhackett1024) [reply src/install.js:90]: This isn't considered an error state --- replay-recordings update-browsers only updates browsers which are already installed, and doesn't download ones which haven't been installed by any packages on the system.  I figured users can use whatever packages they're using (playwright, puppeteer, cypress, more in the future) to download whichever replay browsers they need, and then can use replay-recordings update-browsers to update those without having to use any awkward workarounds.

Comment (ryanjduffy) [reply src/install.js:90]: Seems like if the user passed an invalid directory (e.g. by mistake) for update, we'd do nothing and the user wouldn't know. Am I reading the code incorrectly?

Comment (bhackett1024) [reply src/install.js:90]: You're right, maybe we should count the number of browsers updated and exit with an error if none were found?

**Pull request merged**